source/slang/core.meta.slang

```diff
@@ -1114,6 +1114,14 @@ __generic<E : __EnumType>
 __intrinsic_op($(kIROp_Eql))
 bool operator==(E left, E right);
 
+// Binding Attributes
+
+__attributeTarget(DeclBase)
```

Comment:
We might be able to scope this down more tightly to `__attributeTarget(VarDeclBase)`. We'd need to double-check that before we commit to it, though.